backend/src/Functions/Positions/CreatePosition.cs

```diff
@@ -27,6 +27,12 @@ public async Task<IActionResult> Run(
 
         var (error, requestPosition) = await TryDeserializeRequestBody<Position>(req);
 
+        // Check that the fill percentage is not greater than the vacancy percentage
+        if (requestPosition.VacancyFill > requestPosition.VacancySize)
```

Comment:
Validations need to be after we check if the deserialization was successful (`if (error is not null)`).  
Here it is not guaranteed that `requestPosition` is non-null.

Reply:
True, didnt notice late at night. Will update this soon